src/index.js

```diff
@@ -105,18 +104,7 @@ export default {
     );
     return snapshotLayout;
   },
-  async getContextMenu (obj, menu) {
-    const app = qlik.currApp(this);
-    const isPersonalResult = await app.global.isPersonalMode();
-    // This check is done because the desktop wrapper blocks downloads.
-    // It also blocks this feature in QCS currently as isPersonalMode returns true
-    if (
-      !this.$scope.layout.allowexportxls ||
```

Comment:
I don't think the check of `!this.$scope.layout.allowexportxls` should be removed

Reply:
You are right. I shouldn't remove `!this.$scope.layout.allowexportxls`